<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignUp</class>
 <widget class="QDialog" name="SignUp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>823</width>
    <height>568</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QGroupBox" name="login_3">
         <property name="minimumSize">
          <size>
           <width>380</width>
           <height>240</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>MS Reference Sans Serif</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="title">
          <string>Регистрация</string>
         </property>
         <widget class="QPushButton" name="regB">
          <property name="geometry">
           <rect>
            <x>190</x>
            <y>210</y>
            <width>181</width>
            <height>21</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>MS Reference Sans Serif</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Зарегистрироваться</string>
          </property>
         </widget>
         <widget class="QPushButton" name="backMenu">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>210</y>
            <width>171</width>
            <height>21</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>MS Reference Sans Serif</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Назад в меню</string>
          </property>
         </widget>
         <widget class="QWidget" name="layoutWidget_4">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>80</y>
            <width>361</width>
            <height>36</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLabel" name="mail">
             <property name="text">
              <string>mail adress </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="mailText">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>40</y>
            <width>361</width>
            <height>36</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="nameLabel">
             <property name="text">
              <string>Login          </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="nameText">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="layoutWidget_3">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>120</y>
            <width>361</width>
            <height>36</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="password">
             <property name="text">
              <string>Password    </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="passwordText">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="layoutWidget_5">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>160</y>
            <width>361</width>
            <height>36</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QLabel" name="paswordS">
             <property name="text">
              <string>Password x2</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="passwordSText">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
